README.md

```diff
@@ -72,6 +72,12 @@ spec hasn't changed since the last run:
 
 The usual behavior of `run` or `all` is not to run the `api` target if the OpenAPI spec has not been touched/changed.
 
+To access the Go Docs for validation-service run: 
+
+    make godoc
+
+and navigate to `http://localhost:6060/pkg/github.com/UCLALibrary/validation-service/`
```

Comment:
On seeing the site, yes, let's use pkgsite instead for display. It's UI is much better.